src/store/modules/order/actions.ts

```diff
@@ -0,0 +1,47 @@
+import { ActionTree } from 'vuex'
+import store from '@/store'
+import RootState from '@/store/RootState'
+import OrderState from './OrderState'
+import * as types from './mutation-types'
+import router from '@/router'
+
+
+const actions: ActionTree<OrderState, RootState> = {
+  updateOrderList ({ commit }, payload) {
+    commit(types.ORDER_LIST_UPDATED,  payload );
+  },
+  modifyCsv ({ commit }, payload) {
+    commit(types.MODIFY_CSV, payload);
+  },
+  async groupProducts ({commit}, csv: any) {
```

Comment:
I this should a getter instead

Reply:
In this action we are fetching product information and storing in order state.
Can we still add it in getters?